packages/node_modules/@webex/plugin-meetings/src/reconnection-manager/index.js

```diff
@@ -386,7 +387,7 @@ export default class ReconnectionManager {
     PeerConnectionManager.close(meeting.mediaProperties.peerConnection);
     meeting.mediaProperties.unsetPeerConnection();
     meeting.mediaProperties.reInitiatePeerconnection();
-
+    MeetingUtil.setPeerConnectionEvents(meeting);
```

Comment:
Since this is here, do we still need the `TODO` on line 2360 of meeting/index.js?